src/display.rs

```diff
@@ -1,28 +1,65 @@
-use matcher::Matches;
+use matcher::{Matches, Match};
 use std::fmt;
+use colored::*;
 
-pub struct MatchDisplay {
+// MatchDisplay to format a single Match
+pub struct MatchDisplay<'a> {
```

Comment:
In looking at this again, the reason for the initial match display was to have a struct that held the options for displaying. We may not need this data type as there won't be explicit per match options.